backend/indexer/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.22-bullseye AS build
+
+WORKDIR /go/src/pdp-explorer/backend/indexer
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+RUN go build -o /go/bin/pdp-indexer ./cmd/indexer
+
+FROM gcr.io/distroless/cc
```

Comment:
I guess this has been around for ages but I hadn't heard about or really understood it.  Container with no shell is such a good idea.